.circleci/config.yml

```diff
@@ -24,3 +37,44 @@ workflows:
               only:
                 - main
           context: cimg-publishing
+
+jobs:
+  check-feed:
+    docker:
+      - image: cimg/base:2023.01
```

Comment:
shouldn't this be changed to 2023.04 now that we are in April?

Reply:
i do not think this matters since there are no dependencies other than git/shell being used. the goal is to create a dockerfile with everything we need, including the ability to make pull requests